src/react/src/features/auth/authSlice.ts

```diff
@@ -66,33 +89,47 @@ const authSlice = createSlice({
             .addCase(login.fulfilled, (state, action) => {
                 state.isLoggedIn = true;
                 state.status = 'idle';
```

Comment:
```suggestion
                state.status = 'idle';
                state.message= '';
```
Also for some reason if you get an error on login and then go to register the error persists in the register page